docs/references/astro/organization-store.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: $organizationStore
+description: Clerk's $organizationStore store retrieves the currently active organization.
+---
+
+# `$organizationStore`
+
+The `$organizationStore` store is used to retrieve attributes of the currently active organization.
```

Comment:
Are there plans to add the `useOrganizationList()` type info here?

Reply:
If that means implementing pagination as well, not sure @panteliselef

Reply:
If we can't offer up everything in `useOrganizationList()`, at least initially, we can also just call that out here so developers know. Better to say something isn't supported that leave them hunting for how to do it.

Reply:
Thanks @royanger. I'll apply that mindset to other SDKs as well (Vue, Svelte) :)

Reply:
Awesome!